icons/graylog.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Graylog icon</title><path d="M7.487 11.311a.84.84 0 01.748.45h.706l1.112-2.674a.483.483 0 01.3-.278c.235-.043.47.085.513.32l1.177 5.177 1.198-6.973a.41.41 0 01.32-.343.44.44 0 01.535.321l1.284 5.24.663-1.946a.449.449 0 01.17-.235c.193-.128.471-.085.6.107l.556.791c.021.193.021.385.021.578a8.3 8.3 0 01-.042.749c-.086-.022-.15-.086-.214-.15l-.556-.77-.856 2.588a.448.448 0 01-.556.278.393.393 0 01-.278-.3L13.733 9.58l-1.22 7.08a.449.449 0 01-.492.363c-.192-.02-.32-.17-.363-.363l-1.305-5.99-.706 1.69a.439.439 0 01-.406.278H8.235a.863.863 0 01-.748.428.88.88 0 01-.877-.877c.021-.47.406-.877.877-.877zM12 .396c6.973 0 12 5.369 12 11.615 0 6.353-4.77 11.593-12 11.593S0 18.364 0 12.011C-.02 5.765 5.005.396 12 .396zM4.064 12.01c0 4.256 3.658 8 7.915 8 4.256 0 7.914-3.744 7.914-8 0-4.6-3.658-8.043-7.914-8.043-4.236 0-7.915 3.444-7.915 8.043z"/></svg>
```

Comment:
Hi @loganmarchione - thanks for your contribution! 🎉
Just having a quick look, it appears the inner 'pulse' on your PR (red) has been stretched out. Any chance you could take another look at it?
![image](https://user-images.githubusercontent.com/1830867/103996792-3e771a00-5192-11eb-9701-a81374c57f4e.png)